File: array/reverseArray.py
def reverseArray(arr):
    start=0
    end=len(arr)-1
    while(start<end):
        temp=arr[start]
        arr[start]=arr[end]
        arr[end]=temp
        start+=1
        end-=1
    
    return arr
    
print(reverseArray([1,2,3,4,6]))

File: array/reverseArray.js
// using two pointer method
function reverseArray(arr) {
  // two pointer if.e start and end
  let start = 0;
  let end = arr.length - 1;

  while (start < end) {
    let temp = arr[start];
    arr[start] = arr[end];
    arr[end] = temp;
    start++;
    end--;
  }
  return arr;
}

console.log(reverseArray([1, 2, 3, 4, 5]));
